README.md

```diff
@@ -1,54 +1,104 @@
+        <img src="https://godoc.org/github.com/kwilteam/kwil-db?status.svg" alt="GoDoc">
+    </a>
+    <a href="https://goreportcard.com/report/github.com/kwilteam/kwil-db">
+        <img src="https://goreportcard.com/badge/github.com/kwilteam/kwil-db" alt="Go Report Card">
+    </a>
+    <a href="https://discord.com/invite/HzRPZ59Kay">
+        <img alt="Discord" src="https://img.shields.io/discord/819855804554543114?logo=discord">
+    </a>
+</div>
+
+Kwil-db is the execution layer (node software) for Kwil Networks. Built with [PostgreSQL](https://www.postgresql.org/) and [CometBFT](https://github.com/cometbft/cometbft), Kwil-db extend the functionality of traditional relational databases to enable secure, byzantine fault tolerant, relational data-driven replicated state machines.
 
 ## Overview
 
-Learn more about Kwil at [kwil.com](https://kwil.com).
+To learn more about high-level Kwil concepts, refer to the [Kwil documentation](https://docs.kwil.com/docs/concepts).
+
+To test deploying and using a decentralized database on the Kwil testnet, refer to the [Kwil testnet tutorial](https://docs.kwil.com/docs/testnet/quickstart).
+
+For more information on kwil-db, check out the [Kwil node documentation](https://docs.kwil.com/docs/node/quickstart).
+
+## Quickstart
 
-## Build instructions
+### Build Instructions
 
-### Prerequisites
+#### Prerequisites
 
 To build Kwil, you will need to install:
 
 1. [Go](https://golang.org/doc/install)
 2. [Protocol Buffers](https://protobuf.dev/downloads/) (optional), with the `protoc` executable binary on your `PATH`.
 3. [Taskfile](https://taskfile.dev/installation)
-4. Protocol buffers go plugins and other command line tools. The `tool` task will install the required versions of the tools into your `GOPATH`, so be sure to include `GOPATH/bin` on your `PATH`.
+4. [Docker](https://docs.docker.com/get-docker/) to run a PostgreSQL database.
+5. Miscellaneous go plugins and other command line tools. The `tool` task will install the required versions of the tools into your `GOPATH`, so be sure to include `GOPATH/bin` on your `PATH`.
 
     ```shell
     task tools
     ```
 
-### Build
+#### Build
 
-Invoke `task` command to see all available tasks. The `build` task will compile `kwild`, `kwil-cli`, and `kwil-admin`. They will be generated in `.build/`:
+Invoke `task` command to see all available tasks. The `build` task will compile `kwild`, `kwil-cli`, and `kwil-admin` binaries. They will be generated in `.build/`:
 
 ```shell
 task build
 ```
 
-## Local deployment
+### Local Deployment
+
+#### Start PostgreSQL
+
+Each Kwil node requires a PostgreSQL instance to run. You can start a PostgreSQL database using Docker Compose:
+
+```shell
+docker compose -f ./deployments/compose/postgres/docker-compose.yml up
+```
+
+### Start kwild
 
 You can start a single node network using the `kwild` binary built in the step above:
 
 ```shell
-.build/kwild --autogen
+.build/kwild --autogen --app.pg_db_host localhost
```

Comment:
It's time for `go run github.com/kwilteam/kwil-db/cmd/kwild@v0.7.0-beta --autogen` since we opensourced the repo,  but we need some changes to make this work